Braintree/Drop-In/BTDropInViewController.m

```diff
@@ -652,6 +652,72 @@ - (void)fetchPaymentMethods {
     }];
 }
 
+#pragma mark - Card Properties
+
+- (NSString *)cardNumber {
+    return self.dropInContentView.cardForm.number;
+}
+
+- (void)setCardNumber:(NSString *)cardNumber {
+    self.dropInContentView.cardForm.number = cardNumber;
+}
+
+- (NSString *)cardExpirationMonth {
+    return self.dropInContentView.cardForm.expirationMonth;
+}
+
+- (NSString *)cardExpirationYear {
+    return self.dropInContentView.cardForm.expirationYear;
+}
+
+- (void)setCardExpirationDate:(NSDate *)expirationDate {
+    [self.dropInContentView.cardForm setExpirationDate:expirationDate];
+}
+
+- (NSString *)cardCVV {
+    return self.dropInContentView.cardForm.cvv;
+}
+
+- (void)setCardCVV:(NSString *)cardCVV {
+    self.dropInContentView.cardForm.cvv = cardCVV;
+}
+
+- (BOOL)requireCardCVV {
+    return self.dropInContentView.cardForm.optionalFields & BTUICardFormOptionalFieldsCvv;
+}
+
+- (void)setRequireCardCVV:(BOOL)requireCardCVV {
+    if (self.requireCardCVV != requireCardCVV) {
+        self.dropInContentView.cardForm.optionalFields ^= BTUICardFormOptionalFieldsCvv;
```

Comment:
Open to suggestions, but this seems simple and idiomatic for toggling a bit. :smile: